operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3889,11 +3889,8 @@ continuous_test:
   enabled: false
   # -- Number of replicas to start of continuous test
   replicas: 1
-  image:
-    repository: grafana/mimir-continuous-test
-    tag: 2.12.0
-    pullPolicy: IfNotPresent
-    # Note: optional pullSecrets are set in toplevel 'image' section, not here
```

Comment:
can you also remove this line https://github.com/grafana/mimir/blob/ddbb42c5d96c07d017a40d784c3aaed2a7caa158/.github/workflows/scripts/helm-weekly-release.sh#L103

Reply:
done, updated continuous_test as well, now testing, but pushed early.